scss/_card.scss

```diff
@@ -0,0 +1,92 @@
+.card{
+    width: 340px;
+    height: 355px;
```

Comment:
No a good idea to restrict the height of blocks with text, this is what can happen when there's too much text. You can use min-height instead.
Also note how the image shrank.

<img width="629" alt="Screenshot 2024-08-23 at 12 24 46 PM" src="https://github.com/user-attachments/assets/09928b46-a02f-44dc-ab8a-fe660fb91394">